README.md

```diff
@@ -1 +1,141 @@
+
+
+## 🛠 Built With <a name="built-with">HTML/CSS</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML/CSS</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+Understand how to:
+Use Git and Github.
+Use HTML to build websites.
+Use CSS to style web pages.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+Install Git,
+Install npm (node.js),
+Have a working browser,
+Have a strong internet connection,
+Install VSCode with with extentions like live server, prettier formatter etc,
+Have a laptop as well
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  git clone git@github.com:ismailhallirumuhammad/hello-world.git
+
+### Install
+
+Install this project with:
+
+Linters and there are many ways you can integrate a linter they are but not limited to the following:
+
+1-Text editor plugins,
+2-GitHub Actions,
+3-GitHub apps,
+Instructions:
+a-Make sure that there are no linter errors:
+b-Linter should be set up correctly for your repository.
+c-Linter result should be green.
+d-There should be no custom changes to linter config.
+
+### Usage
+
+To run the project, execute the following command:
+You can open/run this project using or by lunching the live server plugging in VSCode.
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/ismailhallirumuhammad)
+- Twitter: [@twitterhandle](https://mobile.twitter.com/IsmailhalliruM1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/mwlite/in/ismail-halliru-muhammad-2a8453127)
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ismailhallirumuhammad/hello-world/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, you can contact the owner or use it for referrence...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank:
+The entire Microverse team for their endless effort most especially the CEO,
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- Great Job for adding the `mit` license, I think this note is only to suggest to use `mit` license, so remove the note to have a professional readme 👌